### PR TITLE
[staging] python3Packages.paramiko: 2.7.1 -> 2.7.2, python3Packages.sshtunnel: 0.3.1 -> 0.4.0

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
Updating packages I maintain

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/paramiko/default.nix [hunk 1]
```diff
@@ -14,20 +14,13 @@
...
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "920492895db8013f6cc0179293147f830b8c7b21fdfc839b6bad760c27459d9f";
+    sha256 = "7f36f4ba2c0d81d219f4595e35f70d56cc94f9ac40a6acdf51d6ca210ce65035";
   };
 
-  patches = [
-    # fix RSA key loading with cryptography 3.1, remove >2.7.1
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/paramiko/default.nix:26]: I think fetchpatch can be removed from inputs

Comment (jonringer) [reply pkgs/development/python-modules/paramiko/default.nix:26]: done

**Comment by jonringer** [comment]
@GrahamcOfBorg build azure-cli

**Comment by SuperSandro2000** [comment]
If it is only 100 to 500 rebuilds we could also merge it into master.

But building staging takes to long with my machine.

File: pkgs/development/python-modules/paramiko/default.nix [hunk 2]
```diff
@@ -14,20 +13,13 @@
...
-      url = "https://github.com/paramiko/paramiko/commit/81064206bf3cec2ca4372257ff138481e1227b91.patch";
-      sha256 = "01b87ffgyvd6rilp1w1kf7lk29z706ch39nwl21ifklqpjhmazww";
-    })
-  ];
   checkInputs = [ invoke pytest mock pytest-relaxed ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/paramiko/default.nix:23]: We could move to pytestCheckHook here at some point.

**Review by SuperSandro2000 — APPROVED** [review]

File: pkgs/development/python-modules/paramiko/default.nix (hunk 2 above)
Comment (jonringer) [reply pkgs/development/python-modules/paramiko/default.nix:23]: sure, if there's issues when updating again, we can revist this, but I don't think it's harming anything as tests are being ran.

**Pull request merged**